applications/nrf_desktop/doc/info.rst

```diff
@@ -0,0 +1,42 @@
+.. _info:
+
+Info module
+###########
+
+Use the Info module to:
+
+* handle the role of the final subscriber for the :ref:`config_channel` events,
+* provide the board name through the :ref:`config_channel`.
+
+Module Events
+*************
+
++------------------+--------------------------------+---------------+------------------------+------------------+
+| Source Module    | Input Event                    | This Module   | Output Event           | Sink Module      |
++==================+================================+===============+========================+==================+
+| :ref:`hids`      | ``config_event``               |  ``info``     |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               |                        |                  |
+| :ref:`hids`      | ``config_fetch_request_event`` |               |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               +------------------------+------------------+
+|                  |                                |               | ``config_fetch_event`` | :ref:`hids`      |
+|                  |                                |               |                        +------------------+
+|                  |                                |               |                        | :ref:`usb_state` |
++------------------+--------------------------------+---------------+------------------------+------------------+
+
+Configuration
+*************
+
+The Info module is required by the :ref:`config_channel`.
```

Comment:
I think this should be mentioned in the first paragraph, line 5 after the intro.

Reply:
Ok, I will move this part of description.